unit_tests/util/serialization.cpp

```diff
@@ -0,0 +1,38 @@
+#include "storage/serialization.hpp"
```

Comment:
Good idea. We should add tests for the other serialization methods too at some point.